src/Estimators/include/BipedalLocomotion/FloatingBaseEstimators/FloatingBaseEstimator.h

```diff
@@ -201,6 +208,20 @@ class FloatingBaseEstimator : public BipedalLocomotion::System::Advanceable<Floa
     bool setKinematics(const Eigen::VectorXd& encoders,
                        const Eigen::VectorXd& encoderSpeeds);
 
+    /**
+     * Set the relative pose of a landmark relative to the base link
+     *
+     * @param[in] landmarkID unqiue landmark ID
```

Comment:
```suggestion
     * @param[in] landmarkID unique landmark ID
```

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/254/commits/10e3009d2a596e5fe5285718b91c776db4683086